acceptance/tests/direct_puppet/supports_utf8.rb

```diff
@@ -50,18 +53,18 @@
   step 'run agent(s)' do
     with_puppet_running_on(master, {}) do
       agents.each do |agent|
-        fqdn = agent.hostname
+        fqdn = agent_to_fqdn(agent)
         config_version = ''
         config_version_matcher = /configuration version '(\d+)'/
-        on(agent, puppet("agent -t --environment #{tmp_environment} --server #{master.hostname}"),
+        on(agent, puppet("agent -t --environment '#{tmp_environment} --server #{master.hostname}'"),
```

Comment:
yeah, the single quote should just be on the tmp environment path:  '#{tmp_environment}'